### PR TITLE
new: AwaitingConfirmationPage Component

### PR DESCRIPTION
**Pull request by dougfabris**
- [x] I have read the Contributing Guide
- [x] I have signed the CLA
- [x] Lint and unit tests pass locally with my changes
- [x] I have labeled the PR correctly with the related package
- [x] I have run Loki's visual regression tests (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/onboarding-ui/.storybook/preview.tsx [hunk 1]
```diff
@@ -38,10 +38,9 @@ const getI18n = () => {
             awaitingConfirmation: {
               title: 'Awaiting confirmation',
               subtitle:
-                'We have sent you an email to emailname@email.com with a confirmation link. Please verify that the security code below matches the one in the email.',
-              code: 'Funny Tortoise In The Hat',
+                'We have sent you an email to {{emailAddress}} with a confirmation link. Please verify that the security code below matches the one in the email.',
               codeFallback:
-                'Didn’t receive email? <0>Resend</0> or <1>Change email</1>',
```
Comment (dougfabris) [inline packages/onboarding-ui/.storybook/preview.tsx]: @tassoevan why?

File: packages/onboarding-ui/.storybook/preview.tsx [hunk 2]
```diff
@@ -38,10 +38,9 @@ const getI18n = () => {
...
+                'Didn’t receive email? <1>Resend</1> or <3>Change email</3>',
```
Comment (dougfabris) [inline packages/onboarding-ui/.storybook/preview.tsx]: @tassoevan why?

File: packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx [hunk 3]
```diff
@@ -50,13 +64,17 @@ const AwaitingConfirmationPage = (): ReactElement => {
...
           <Box fontScale='c1'>
             <Trans i18nKey='page.awaitingConfirmation.codeFallback'>
-              <Link href='#' />
-              <Link href='#' />
+              Didn’t receive email?
+              <ActionLink onClick={onResendEmailRequest}>Resend</ActionLink>
+              or
+              <ActionLink onClick={onChangeEmailRequest}>
+                Change email
+              </ActionLink>
```
Comment (dougfabris) [inline packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx:77]: it's necessary to have the text inside of the component?

File: packages/onboarding-ui/.storybook/preview.tsx (hunk 2 above)
Comment (tassoevan) [reply packages/onboarding-ui/.storybook/preview.tsx]: It allows us to show how the text is structured in the JSX too.

File: packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx (hunk 3 above)
Comment (tassoevan) [reply packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx:77]: Not really, but without it is hard to guess how it should be structured. It's very different from just replacing a string, when a key is enough.

**Review by tassoevan — APPROVED** [review]

File: packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx (hunk 3 above)
Comment (dougfabris) [reply packages/onboarding-ui/src/pages/AwaitingConfirmationPage/AwaitingConfirmationPage.tsx:77]: make sense, thanks!

File: packages/onboarding-ui/.storybook/preview.tsx (hunk 2 above)
Comment (dougfabris) [reply packages/onboarding-ui/.storybook/preview.tsx]: got it, thank you!

**Pull request merged**